src/blocks/folder-blocks/summarize/index.tsx

```diff
@@ -0,0 +1,256 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import SyntaxHighlighter from "react-syntax-highlighter";
+import { FileBlockProps, getLanguageFromFilename } from "@githubnext/utils";
+import "./index.css";
+
+export default function (props: FileBlockProps) {
+  const { content, context } = props;
+  const [sections, setSections] = useState<any[]>([]);
```

Comment:
Why `any[]`? Don't we have a type definition for a section further down in the file?